### DISCUSSION TITLE
ring-opacity not working with css var colors

### DISCUSSION THREAD

**Discussion by stefan-schweiger** [tailwindlabs/tailwindcss#4205] [Help]
When I try to use a custom color which has a css var value tailwind isn't able to handle the ring opacity (and I'm guessing similar opacity utilities) correctly.

I guess this is because it doesn't know how to convert `var(--color-primary)` to something like `rgba(0, 255, 0, 0.5)`.
But it there a way to configure a css var color so tailwind can get the required information?

It would be great if it would be able to understand something like `rgb(var(--color-primary-rgb))` (with `--color-primary-rgb: 0, 255, 0;`) and pick it up from there to create `rgba(var(--color-primary-rgb), 0.5)`. But sadly this doesn't work.

https://play.tailwindcss.com/I7Z5VDUPf7

**Comment by yabdab** [comment]
I am having same issue, but my config is a bit different. ring seems to be the only color that does not get opacity applied?

https://play.tailwindcss.com/tPlBMUMNKx?file=config

**Reply by stefan-schweiger** [reply]
I've created a PR to make this a bit easier, but it hasn't been merged, but in this comment you can see a "workaround" https://github.com/tailwindlabs/tailwindcss/pull/4659#issuecomment-876299918

But you need to store the colors as rgb values like `--color-primary-rgb: 255, 127, 0`
